### DISCUSSION TITLE
Allow using Docker Compose overrides with git

### DISCUSSION THREAD

**Discussion by peaklabs-dev** [coollabsio/coolify#4339] [Feature Requests]
When deploying via git and docker compose - overrides should be possible and the UI should show the final compose. Also set an option to add overrides file path and make an option to set different overrides per environment.

**Comment by Melodeiro** [comment]
Being able to edit it directly within a resource configuration, such as when editing the 'empty Docker Compose' template, could also be very convenient.
![image](https://github.com/user-attachments/assets/acbb97de-0314-4bd5-a1dc-a37249eb614b)
PS: After working with that feature i really like it, because you can close ports with it, disable services, change envs, etc, without modifying the files of a public repo

**Comment by madfcat** [comment]
How is it not a thing yet?
I want to have `docker-compose.yaml` and `docker-compose.override.yaml` for my production build.

**Comment by Melodeiro** [comment]
Yeah, right now we have to fork/clone a 3-rd party repo and change the docker compose file to being able to change something

**Comment by Squix** [comment]
+1 this

**Comment by ahmtgrbz** [comment]
I really need too.

**Comment by silva-guimaraes** [comment]
Bump

**Comment by luishugoooo** [comment]
+1

**Comment by aderler** [comment]
+1 we also need this

**Comment by pedroricha** [comment]
+1
